lib/hyrax/hirmeos/work_factory.rb

```diff
@@ -5,12 +5,23 @@ class Hyrax::Hirmeos::WorkFactory
   def self.for(resource:)
     work = Hyrax::Hirmeos::Client::Work.new
     work.title = resource.title
-    work.uri = [{ uri: resource_url(resource), canonical: true }]
+    work.uri = [{ uri: resource_url(resource), canonical: true },
+                { uri: resource.id }]
     work.type = "other" # Need to map Hyrax work type to configured work types in HIRMEOS
+    file_urls = add_file_urls(resource)
+    work.uri << file_urls if file_urls.present?
+    work.uri.flatten!
     work
   end
 
   def self.resource_url(work)
     Rails.application.routes.url_helpers.polymorphic_url(work)
   end
+
+  def self.add_file_urls(work)
+    files = work.file_sets
+    return if files.blank?
+    links = files.map { |file| Hyrax::Engine.routes.url_helpers.download_url(id: file, locale: 'en') }
```

Comment:
I wonder if adding the `locale` here is unnecessary and potentially problematic if HIRMEOS is watching for an exact uri match and the user has a different locale set.

Reply:
Yeah might not be needed, Ill remove